FirebaseAdmin/FirebaseAdmin/Auth/Multitenancy/Tenant.cs

```diff
@@ -13,7 +13,6 @@
 // limitations under the License.
 
 using Google.Apis.Util;
```

Comment:
Just wondering, is this import being used in this file?

Reply:
Yes, we use some of its extension methods.